web/client/src/App.js

```diff
@@ -22,25 +27,53 @@ function App() {
       <Grid item xs>
         <Grid container direction="column">
           <MaterialUI paperClass="paper-top">
-            <TokenForm parentCallback={callBackToken} />
+            <ModuleInfo
```

Comment:
I think we should also mention that a JWT Token can only be obtained with a service account key

Reply:
Also the command info does not work with a JWT token so that should be mentioned as well

Reply:
Will put that into the token validation part